Source/Scene/PointCloudEyeDomeLighting.js

```diff
@@ -146,21 +107,12 @@ function createCommands(processor, context) {
 }
 
 function createResources(processor, context) {
-  var screenWidth = context.drawingBufferWidth;
-  var screenHeight = context.drawingBufferHeight;
-  var colorGBuffer = processor._colorGBuffer;
+  var width = context.drawingBufferWidth;
+  var height = context.drawingBufferHeight;
   var nowDirty = false;
-  var resized =
-    defined(colorGBuffer) &&
-    (colorGBuffer.width !== screenWidth ||
-      colorGBuffer.height !== screenHeight);
-
-  if (!defined(colorGBuffer) || resized) {
-    destroyFramebuffer(processor);
-    createFramebuffer(processor, context);
-    createCommands(processor, context);
-    nowDirty = true;
-  }
+  processor._framebuffer.update(context, width, height);
+  createCommands(processor, context);
+  nowDirty = true;
   return nowDirty;
```

Comment:
`nowDirty` will be true every frame, which will cause a bunch of extra work in `update`.

Looking at this closer, `nowDirty` and `dirty` could be removed entirely since `derivedCommand.framebuffer !== this.framebuffer` should cover the bases now. Previously this was not the case.

One way to ensure that eye dome lighting still works is to run through the closed issues in https://github.com/CesiumGS/cesium/pull/9719 and make sure this PR hasn't introduced any regressions.

Reply:
Checked the local Sandcastles in the PR and didn't run into any issues after removing `nowDirty` and the return statement in `createResources`. Is this possible because FramebufferManager doesn't recreate the framebuffer when `nowDirty` used to be false?

Reply:
> Is this possible because FramebufferManager doesn't recreate the framebuffer when `nowDirty` used to be false?

Yeah. The opposite is true too, FramebufferManager will recreate the framebuffer when `nowDirty` used to be true.